web/tasks/DART.md

```diff
@@ -10,110 +11,175 @@ motivation: TODO.
 
-[More Information Needed]
+## Dataset Creation 
 
-### Communicative Goal
+### Curation Rationale
+
+The dataset creators encourage through DART further research in natural language generation from semantic data.
+DART provides high-quality sentence annotations with each input being a set of entity-relation triples in a tree structure. 
 
-[More Information Needed]
+### Communicative Goal 
 
+The speaker is required to produce coherent sentences and construct a trees structured ontology of the column headers.
 
 ### Source Data
 
+The dataset re-uses data from the following pre-existing resources:
+(1) human annotation on open-domain Wikipedia tables from WikiTableQuestions ([Pasupat and Liang,
+2015](https://www.aclweb.org/anthology/P15-1142.pdf)) and WikiSQL ([Zhong et al., 2017](https://arxiv.org/pdf/1709.00103.pdf))
+(2) automatic conversion of questions in
+WikiSQL to declarative sentences
+(3) incorporation of existing datasets including WebNLG 2017 (Gardent et al., 2017[a](https://www.aclweb.org/anthology/P17-1017.pdf),[b](https://www.aclweb.org/anthology/W17-3518.pdf); [Shimorina and Gardent, 2018](https://www.aclweb.org/anthology/W18-6543.pdf)) and Cleaned E2E ([Novikova et al., 2017b](https://arxiv.org/pdf/1706.09254.pdf); Dušek et al., [2018](https://arxiv.org/pdf/1810.01170.pdf), [2019](https://www.aclweb.org/anthology/W19-8652.pdf))
+
+Creators also explored automatic alignments between the knowledge base and text including Neural Wikipedian ([Vougiouklis et al., 2018](https://arxiv.org/pdf/1711.00155.pdf)) and TRex ([Elsahar et al., 2018](https://www.aclweb.org/anthology/L18-1544.pdf)).
+
+We refer the reader to the papers describing these sources for further information.
+
 #### Initial Data Collection and Normalization
 
-[More Information Needed]
+The training data consists of concept sets and captions for the source datasets listed above.
+For conversion of a meaning representation (MR) to a triple set, where the NAME slot was represented as the subject.
 
 #### Who are the source language producers?
 
-[More Information Needed]
+Human annotations and an automatic conversion procedure.
 
 ### Annotations
 
 #### Annotation process
 
-[More Information Needed]
+We propose a two-stage annotation process for constructing triple set sentence pairs based on a tree-structured ontology
+of each table. First, internal skilled annotators denote the parent column for each column header.
+Then, a larger number of annotators provide a sentential description of an automatically-chosen subset of table cells in a row. To form a triple set sentence pair, the highlighted cells can be converted to a connected triple set automatically according to the column ontology for the given table.
 
 #### Who are the annotators?
 
-[More Information Needed]
+Internal annotators from the research group ([Figure 7](https://arxiv.org/pdf/2007.02871.pdf)) and Amazon Mechanical Turk (MTurk) workers ([Figure 9](https://arxiv.org/pdf/2007.02871.pdf)).
```

Comment:
Could you note whether more information is provided? https://gem-benchmark.github.io/tasks/guide#who-are-the-annotators 

It is fine to just write a sentence "No further information about the MTurk workers has been provided".